config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gcp-gpu-upgrade-downgrade.yaml

```diff
@@ -178,65 +178,4 @@ periodics:
 
   annotations:
     testgrid-dashboards: google-gce-upgrade
-    testgrid-tab-name: gce-gpu-1.13-master-master-upgrade
-- cron: "0 10-23/12 * * *"
```

Comment:
Why we're removing these?

Reply:
because the upgrade tests are the priority, and the downgrade tests are solidly failing (so no one is paying attention to the signal from them)